app/controllers/vm_infra_controller.rb

```diff
@@ -2,6 +2,7 @@ class VmInfraController < ApplicationController
   include VmCommon # common methods for vm controllers
   include VmRemote # methods for VM remote access
   include VmShowMixin
+  include Mixins::Actions::VmActions::Transform
```

Comment:
@lgalis , is this line needed? I'd say that the mixing is already included in ApplicationController throught CiProcessing and this controller inherits from Application controller.